scripts/install_pecan.sh

```diff
@@ -362,6 +362,9 @@ EOF
   rm /tmp/pecan.conf
 fi
 
+# adding the permissions to www-data to execute bash scripts which need root privilages
```

Comment:
I tried the VM user password change with the bash script but didn't worked out, 
I remove this permissions